ra/ra.go

```diff
@@ -1806,6 +1810,8 @@ func (ra *RegistrationAuthorityImpl) NewOrder(ctx context.Context, req *rapb.New
 			continue
 		}
 
+		// Delete the authz from the nameToExistingAuthz map since we are not reusing it.
```

Comment:
> I think it would be better, on line 1852, to iterate over order.Authorizations.

That doesn't work because `order.Authorizations` are just IDs.

> And actually, a little better than that would be to factor our a function minExpiry that takes an Order and a list of Authorizations, and returns a dat

That's closer to what I was doing before but a pain in `NewOrder`. There is no one collection of `Authorzation` objects to send away.